### ISSUE TITLE
Mobile: login screen doesnt scroll on small screens so cant complete the log in form and use the app

### ISSUE THREAD

**Issue by Baggy8** [bitwarden/mobile#2280]
### Steps To Reproduce

Cannot log into app on phone with small screen because login screen does not scroll. On my telephone I can see the email field, and the "remember me" slider and nothing underneath but it wont let me scroll the screen down so I cant complete the form and log in.

### Expected Result

The screen should be able to scroll so people with small screens can use the app

### Actual Result

Cannot complete form and therefore cannot use the app

### Operating System

Android

### Operating System Version

11 Go

### Device

CAT s22

### Build Version

Dec 1 2022 release on Play store. Cant tell proper version without logging in.

### Beta

- [ ] Using a pre-release version of the application.

**Comment by Squelch** [comment]
I'd like to confirm this issue.

Device: Acer Holo360
Screen Size: 3"
OS: Android 7.1.1
BW Version 2022.11.0(5580)

Using the touch keyboard to enter the email address, the user is unable to progress by pressing enter/return/send key. In the case of a 3" screen, there's only enough to show the email address entry field with nothing accessible via the default touch keyboard below that.

A workaround is to install a different keyboard app that provides navigation keys, or use an external keyboard (BT, USB) and press the down arrow key to move to the next GUI element. Once, and then right to enable "Remember" and then again to highlight the commit button. This also requires that the user has familiarity with the app to do this blind of course.

There are two possible solutions to this:
1. Commit the entered email address upon keyboard enter/return/send by default. This will bypass the "Remember" control for which there does not appear to be an alternative elsewhere.
2. Provide scrolling functionality to the login screen allowing access to all screen controls

**Comment by Squelch** [comment]
> This should be fixed in the 2023.1.0 release currently rolling out (slowly), and the beta channel (available now).

I joined the beta programme, uninstalled and reinstalled the 2023.1.0 release - to clear existing user id persistence

The user can now press enter to continue after entering their email address. "Remember" toggle has been moved to just below the password field and fits within a 3" screen.
I also noticed other improvements with the layout. ie. while adding a pin code.

I am confident that this issue has been resolved satisfactorily, and may be closed.
